plugin/trino-iceberg/pom.xml

```diff
@@ -327,6 +427,18 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>http-server</artifactId>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>io.airlift</groupId>
+                    <artifactId>node</artifactId>
```

Comment:
What's the reason for the exclusion?

Reply:
I think this was necessary as we were iterating through various test configurations, but good callout because I was able to back and reorganize some of the dependencies and it should be cleaner now.